processing/src/main/java/org/apache/carbondata/processing/util/CarbonLoaderUtil.java

```diff
@@ -426,10 +426,10 @@ public static boolean isValidEscapeSequence(String escapeChar) {
   }
 
   public static boolean isValidBinaryDecoder(String binaryDecoderChar) {
-    return binaryDecoderChar.equalsIgnoreCase(
-        CarbonLoadOptionConstants.CARBON_OPTIONS_BINARY_DECODER_BASE64) ||
-        binaryDecoderChar.equalsIgnoreCase(
-            CarbonLoadOptionConstants.CARBON_OPTIONS_BINARY_DECODER_HEX) ||
+    return CarbonLoadOptionConstants.CARBON_OPTIONS_BINARY_DECODER_BASE64.equalsIgnoreCase(
```

Comment:
Why change the order, doesn't make any difference.

Reply:
it will throw NPE if binaryDecoderChar is null. it should return false.